src/pkg/plumbing/tls.go

```diff
@@ -97,18 +97,24 @@ func NewTLSHTTPClient(cert, key, ca, commonName string) *http.Client {
 		log.Panicf("failed to load API client certificates: %s", err)
 	}
 
+	keepAlive := 30 * time.Second
+	if disableKeepAlives {
+		keepAlive = -1 * time.Second
+	}
+
 	transport := &http.Transport{
 		Proxy: http.ProxyFromEnvironment,
 		DialContext: (&net.Dialer{
 			Timeout:   30 * time.Second,
-			KeepAlive: 30 * time.Second,
+			KeepAlive: keepAlive,
```

Comment:
I'm not convinced there's any value to changing the TCP keep-alive settings. Can you please elaborate?

My understanding is that, by setting `DisableKeepAlives`, the HTTP server (Cloud Controller) will close the TCP connection after sending an HTTP response.

Reply:
I think changing the TCP keep-alive settings was a mistake. I removed it from this PR.